google-cloud-bigtable/src/main/java/com/google/cloud/bigtable/data/v2/stub/metrics/Util.java

```diff
@@ -15,20 +15,31 @@
  */
 package com.google.cloud.bigtable.data.v2.stub.metrics;
 
+import com.google.api.gax.rpc.ApiCallContext;
 import com.google.api.gax.rpc.ApiException;
 import com.google.api.gax.rpc.StatusCode;
 import com.google.api.gax.rpc.StatusCode.Code;
+import com.google.common.collect.ImmutableMap;
+import io.grpc.Metadata;
 import io.grpc.Status;
 import io.grpc.StatusException;
 import io.grpc.StatusRuntimeException;
 import io.opencensus.tags.TagValue;
+import java.util.Arrays;
+import java.util.List;
+import java.util.Map;
 import java.util.concurrent.CancellationException;
 import java.util.concurrent.ExecutionException;
 import java.util.concurrent.Future;
 import javax.annotation.Nullable;
 
 /** Utilities to help integrating with OpenCensus. */
 class Util {
+  static final Metadata.Key<String> ATTEMPT_HEADER_KEY =
+      Metadata.Key.of("attempt", Metadata.ASCII_STRING_MARSHALLER);
+  static final Metadata.Key<String> TIMESTAMP_HEADER_KEY =
+      Metadata.Key.of("client-timing", Metadata.ASCII_STRING_MARSHALLER);
```

Comment:
I dont love the name client-timing. How would you feel about `attempt-timestamp`? or `attempt-epoch`?

Reply:
Updated to attempt-epoch. Will update the cl too